src/client/mod.rs

```diff
@@ -612,7 +616,7 @@ impl<T> VaultClient<T>
     ///
     /// [token]: https://www.vaultproject.io/docs/auth/token.html
     pub fn renew(&mut self) -> Result<()> {
-        let res = try!(self.post("/v1/auth/token/renew-self", None, None));
+        let res = try!(self.post::<_, String>("/v1/auth/token/renew-self", None, None));
```

Comment:
I _think_ that the only places the turbofish are necessary is inside of the library, users shouldn't have to use them (I hope)